File: abc.cpp
// GitHub: EntityPlantt/Kattis
#include <iostream>
#include <algorithm>
using namespace std;
int main() {
    int vars[3];
    cin >> vars[0] >> vars[1] >> vars[2];
    sort(vars, vars + 3);
    char c;
    while (cin >> c) {
        cout << vars[c - 'A'] << ' ';
    }
    return 0;
}

File: abc.js
// GitHub: EntityPlantt/Kattis
// Wrong Answer
var vars;
require("readline").createInterface({
	input: process.stdin,
	output: process.stdout
}).on("line", line => {
	if (vars) {
		console.log(Array.from(line).map(elm => vars[elm.charCodeAt(0) - 65]).join(" "));
	}
	else {
		vars = line.split(" ").sort();
	}
});